sormas-api/src/main/java/de/symeda/sormas/api/utils/DataHelper.java

```diff
@@ -345,4 +340,40 @@ public static boolean isParseableInt(String value) {
 			return false;
 		}
 	}
+
+	public static String valueToString(Object value) {
+		Language userLanguage = I18nProperties.getUserLanguage();
+		if (value == null) {
+			return "";
+		} else if (value instanceof Date) {
+			return DateFormatHelper.formatDate((Date) value);
+		} else if (value.getClass().equals(boolean.class) || value.getClass().equals(Boolean.class)) {
+			return DataHelper.parseBoolean((Boolean) value);
+		} else if (value instanceof Set) {
```

Comment:
Not necessarily, since the way element strings are concatenated to get a string for the list is highly dependent on the context, e.g., the delimiter string to use.

Reply:
Fine by me